tests/lwip_sock_dns/tests/01-run.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2022 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from time import sleep
+from testrunner import run
+
+
+def _test_dns_requests(child):
```

Comment:
Can we expect these results to be stable?

Reply:
Maybe we should just check that the IP version is the expected one and then accept any valid address.

Reply:
I also thought about the resolved address of `example.com` and I really don't know about their policy to keep the same IP forever. But we could check for `dns.google.com` instead. I'm pretty sure Google won't give away their nice IPs and the service is also running since 2010. Of course there will be two possible results for IPv4 and IPv6 in this case.
As it goes for the reachability I don't see that this is a huge problem unless you sit behind a great firewall or do I miss something?